infra/messaging/migrations/20190625030841-add-content-hash.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Message', 'contentHash', {
+      type: Sequelize.STRING(66)
```

Comment:
I'd recommend using String rather String(66) in case we change hash function in future.
From DB perspective, there is no storage saving by using String)66).

Reply:
I know this, but I always forget how Postgres does underlying sizing with flexible columns.  Will make this change in a bit.

Reply:
So... apparently Sequelize defaults to varchar(255) if you don't specify a length and I haven't found a way to use a standard postgres style `varchar`.  I'd rather live with a fixed `66` length than a fixed `255` in this case.

Reply:
Oh interesting. I actually did not realize that behavior of sequelize...